README.md

```diff
@@ -85,6 +85,7 @@ This image builds on `ucore-minimal` but adds drivers, storage tools and utiliti
   - all wireless (wifi) card firmwares (CoreOS does not include them) - hardware enablement FTW
   - [mergerfs](https://github.com/trapexit/mergerfs)
   - nfs-utils - nfs utils including daemon for kernel NFS server
+  - [rclone](https://www.rclone.org/) - file synchronization and mounting of cloud storage
```

Comment:
Nitpicky, I know, but would you please move `rclone` below `pcp` for alphabetic order?

(actually, same thing below, in the package list.